docs/markdown/glossary.md

```diff
@@ -0,0 +1,273 @@
+
+A lookup table used by a router to decide who its outgoing receiver should be (the next hop, which can be another router or the destination receiver).                                                                         
+
+## Sender  
+
+The Interledger node that is the initiator (payer) of an Interledger payment.                                                                       
+
+## Settlement  
+
+The finalization of a payment, such that the payee takes the ownership of the transferred funds.                                                                        
+
+## Simple Payment Setup Protocol (SPSP)
+
+An application layer protocol for exchanging payment information - destination account and secret - over HTTPS, between a payee and a payer, to facilitate payment over Interledger. 
+
+SPSP uses the STREAM as transport layer protocol. 
+
+See also: [IL-RFC 9](https://github.com/interledger/rfcs/blob/master/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)                                                                        
+
+## Source (transfer, ledger, amount)   
+
+The transfer/ledger/amount directly adjacent to the sender.                                                                         
+
+## STREAM  
+
+A Transport Protocol designed to be used with ILPv4. Application Layer protocols can use STREAM to send money and data between endpoints.
+
+See also: [IL-RFC 29](https://github.com/interledger/rfcs/blob/master/0029-stream/0029-stream.md)                                                                   
+
+## Streaming Payment   
+
+An ongoing payment where small amounts of money are sent over time to pay for some ongoing service or rent. For instance, a user might pay for a streaming video with a streaming payment. The term "streaming payment" is commonly reserved for cases where the interval is short, from one second up to about a day.                                                                      
+
+## Transaction 
+
+See instead: [Payment](#payment)                                                                     
+
+## Transfer    
+
+The movement of assets from one peer to another. Multiple transfers can be chained together into one multi-hop payment.                                                                         
+
+## Transport Layer 
+
+The transport layer transports payment data between two parties, using the [STREAM](https://github.com/interledger/rfcs/blob/master/0029-stream/0029-stream.md) protocol. Transport Layer is the middle layer of the Interledger protocol stack, which determines the condition and encoding of the data in the ILP Packet. It also determines what details the sender and receiver need to discuss beforehand.                                                                      
+
+## Value   
+
+A financial asset that represents currency. It can be fiat (such as USD, EUR, JPY, etc.) or digital (such as  BTC, ETH, XRP, etc.). 
```

Comment:
I think currency is a subset of "value". Value is harder to define but can also include non-currency assets

Reply:
This definition is with-respect-to  Interledger. I agree that value can include non-currency assets. But currently, I think, in Interledger we can only transfer currency. No?

Reply:
> in Interledger we can only transfer currency. No?

Technically Interledger can transfer any fungible asset. So, imagine: "kernels of corn" or "shares of apple stock" or whatever.